### PR TITLE
Add e2e tests requirements for Beta structured logging

### PR DESCRIPTION
**Pull request by serathius**
/cc @ehashman

### CODE REVIEW

File: keps/sig-instrumentation/1602-structured-logging/README.md [hunk 1]
```diff
@@ -384,6 +384,7 @@ Bring feature quality up to and prevent regression during ongoing migration:
 * JSON format should support same set of feature flags as text format, minus those we will decide to deprecate.
 * Structured logging interface design is verified by migrating one whole component (Kubelet) to structured logging. All other component migrations will be best effort.
 * Log volume (i.e. size of output) will be benchmarked and documented for kubelet in non-JSON and JSON format.
+* Dedicated periodic e2e test scenario with Json enabled
```
Comment (dims) [inline keps/sig-instrumentation/1602-structured-logging/README.md:387]: Json -> JSON?

Comment (serathius) [inline keps/sig-instrumentation/1602-structured-logging/README.md:387]:
```suggestion
* Dedicated periodic e2e test scenario with JSON enabled
```

Comment (serathius) [reply keps/sig-instrumentation/1602-structured-logging/README.md]: done

**Review by ehashman — APPROVED** [review]
Yeah we should do this.

/lgtm
/approve

**Pull request merged**